lib/galaxy/config/__init__.py

```diff
@@ -817,6 +817,7 @@ def _process_config(self, kwargs):
         self.interactivetools_map = self._in_root_dir(kwargs.get("interactivetools_map", self._in_data_dir("interactivetools_map.sqlite")))
         self.interactivetools_prefix = kwargs.get("interactivetools_prefix", "interactivetool")
         self.interactivetools_proxy_host = kwargs.get("interactivetools_proxy_host", None)
```

Comment:
This is not necessary: it's processed automatically

Reply:
I didn't add them, they were here from the original IT work. If I'm understanding correctly, you're suggesting they should all be removed from here and just added to the schema? (at least `interactivetools_prefix` is not in the schema already)